docs/modules/ROOT/pages/how-to/customize-cypher.adoc

```diff
@@ -6,7 +6,7 @@ This page describes how to customize Cypher queries using Cypher Builder.
 Such a scenario could occur if you:
 
 * Need to embed Cypher strings into existing Cypher queries.
-* Use Cypher Builder within a larger Cypher query.
+* Use Cypher Builder within a larger Cypher query that may not use Cypher Builder .
```

Comment:
clarifying: can this also be understood as "When using Cypher Builder to generate part of a larger Cypher query?" 

This is not a suggestion for a change, I just wanted to make sure I understood.

Reply:
This refers to creating a larger query as a string, and using the Cypher Builder for part of it. I'm sure this can be reworded to be clearer